signing/tests/test_secp256k1_signer.py

```diff
@@ -71,6 +77,16 @@ def test_priv_to_public_key(self):
         public_key2 = context.get_public_key(priv_key2)
         self.assertEqual(public_key2.as_hex(), KEY2_PUB_HEX)
 
+    def test_creating_private_keys_from_hex_and_bytes_to_contexts(self):
+        context = create_context("secp256k1")
+        priv_key = context.private_key_from_hex(KEY1_PRIV_HEX)
+        self.assertEqual(context.get_public_key(priv_key).as_hex(),
```

Comment:
This is a bit of a roundabout way to test the PrivateKey was created correctly. I think you can cut the public key and just use the private key's `as_hex` method directly, like in this test above:
https://github.com/hyperledger/sawtooth-core/blob/4c16158fea7de87b066ea550ee8f93b4e03a1b2e/signing/tests/test_secp256k1_signer.py#L56